HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw

```diff
@@ -230,35 +230,35 @@
     <comment>Cancel button text in the dialog asking to log in to Hyper-V VM.</comment>
   </data>
   <data name="WaitForLoginRequest.Description" xml:space="preserve">
-    <value>You have an action item to follow</value>
+    <value>Complete the action items to continue</value>
     <comment>Text describing the purpose of the dialog asking to log in to Hyper-V VM.</comment>
   </data>
   <data name="WaitForLoginRequest.DismissText" xml:space="preserve">
     <value>Dismiss</value>
     <comment>Dismiss button text in the dialog asking to log in to Hyper-V VM after the last attempt.</comment>
   </data>
   <data name="WaitForLoginRequest.LoginRequiredDescriptionText" xml:space="preserve">
-    <value>Please log in to the Hyper-V to continue the set up, once you logged in please click ‘retry’</value>
+    <value>To continue set up, sign into Hyper-V then select 'retry' below</value>

```

Comment:
NIT: 'Please' would be nice

Reply:
This would be a request to PMs and content writers. I am just a messenger here. (I didn't change these strings blindly though. I asked for a review for more strings than I changed here).